src/components/imagePreviewer/index.jsx

```
@@ -0,0 +1,201 @@
+import React, { useEffect, useState } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)